ansible/roles/openshift_sso_app_zabbix/README.md

```diff
@@ -0,0 +1,34 @@
+openshift_sso_app
+=========
+
+Ansible role for adding a Zabbix aggregate item and trigger to the ops-aws-synthetic host
+
+Requirements
+------------
+
+Ansible Modules:
+
+- tools_roles/lib_openshift_3.2
+
+
+Role Variables
+--------------
+
```

Comment:
Be sure and all all your prefixed role variables that need to be passed in to the Role Variables section. You can see an example at https://github.com/openshift/openshift-tools/blob/master/ansible/roles/openshift_sso_app/README.md